.github/renovate.json5

```diff
@@ -1,20 +1,22 @@
 {
   $schema: 'https://docs.renovatebot.com/renovate-schema.json',
   extends: [
-    'config:base',
-    ':dependencyDashboard',
+    'config:recommended',
     ':labels(dependencies)',
     ':maintainLockFilesMonthly', // update non-direct dependencies monthly
-    ':prConcurrentLimit10', // only 10 open PRs at the same time
+    ':prConcurrentLimitNone', // Remove limit for open PRs at any time.
+    ':prHourlyLimit2' // Rate limit PR creation to a maximum of two per hour.
   ],
-  stabilityDays: 3, // Wait 3 days after the package has been published before upgrading it
+  minimumReleaseAge: 3, // Wait 3 days after the package has been published before upgrading it
   // packageRules order is important, they are applied from top to bottom and are merged,
   // meaning the most important ones must be at the bottom, for example grouping rules
   // If we do not want a package to be grouped with others, we need to set its groupName
   // to `null` after any other rule set it to something.
+  dependencyDashboardHeader: 'This issue lists Renovate updates and detected dependencies. Read the [Dependency Dashboard](https://docs.renovatebot.com/key-concepts/dashboard/) docs to learn more. Before approving any upgrade: read the description and comments in the renovate.json5 file.',
+  dependencyDashboardTitle: 'Dependency Dashboard (read renovate.json5 first, before approving any update )',
```

Comment:
I found a way to remind the maintainers to check the renovate config file before approving any update. I've changed the Dependendy Dashboard issue title, and the header text.

Would this be enough for you to allow major updates via the dashboard? Or do you want me to revert to "ignore major updates completely"?

Reply:
I did not see those! This will definitely alleviate the issues I had.

But I think it is redundant to have both the header and the title warn to check the config. Could you please only keep the header and add a link to the `renovate.json5` file?